crates/telio-starcast/src/lib.rs

```diff
@@ -5,6 +5,15 @@
 //! 2. Starcast transport component responsible for multicasting intercepted traffic to the meshnet peers and handling response messages.
 //! 3. Nat is transport's utility used to SNAT the packets comming from multiple peers into one multicast peer address.
 
+use const_format::formatcp;
+
+pub(crate) const IPV4_MULTICAST_NETWORK: &str = "224.0.0.0/4";
+pub(crate) const IPV6_MULTICAST_NETWORK: &str = "ff02::fb/128";
```

Comment:
NIT: as IpNet and IpAddr have constatnts maybe we should construct them in place and avoid parsing in runtime

Reply:
That would be great and I wanted to do this, unfortunately even though those APIs are const they return Result and unwrap is not const. So yeah, we can in theory keep those as `Result<Ipv4Net, PrefixLenError>`, but still we need unwraps in our code so I've decided that strs are cleaner and parsing is not gonna cost us a lot

Reply:
I've create an issue in ipnet repo: https://github.com/krisprice/ipnet/issues/55 and two PRs: https://github.com/krisprice/ipnet/pull/56 https://github.com/krisprice/ipnet/pull/57 that could allow switch to const IpvXNet items in the future.

Reply:
Also for now I've created the ConstIpvXNet wrappers in our utils